### DISCUSSION TITLE
firebaseのサブコレクションに追加する方法がわかりません

### DISCUSSION THREAD

**Discussion by guchio33** [jphacks/C_2213#24] [Q&A]
postコレクションの中に、tagコレクションを追加したいが分からず悩んでおります

**Comment by Rei-Suzuki1729** [comment]
> postコレクションの中に、tagコレクションを追加したいが分からず悩んでおります

自分もv9になってから開発をしっかりとしてないのですが、基本的にはコレクション、ドキュメント、サブコレクションでネスとしますので、
v9だと
コレクションuser、サブコレクションmemos だと
collection(db, 'users', user!.uid, 'memos')
という感じですね
[参考に](https://zenn.dev/hiro__dev/scraps/bfe3ca1757ffae)
